pennylane/pulse/rydberg.py

```diff
@@ -114,6 +115,140 @@ def circuit():
     return HardwareHamiltonian(coeffs, observables, settings=settings)
 
 
+def rydberg_drive(amplitude, phase, detuning, wires):
+    r"""Returns a :class:`ParametrizedHamiltonian` representing the action of a driving laser
+    field with the given amplitude, detuning and phase acting on the given wires
+
+    .. math::
+
+        \frac{1}{2} \Omega(t) \sum_{i \in \text{wires}} (\cos(\phi)\sigma_i^x - \sin(\phi)\sigma_i^y) -
+        \delta(t) \sum_{i \in \text{wires}} \sigma_i^z
+
+    where :math:`\Omega`, :math:`\delta` and :math:`\phi` correspond to the amplitude, detuning
+    and phase of the laser, :math:`i` correspond to the wire index, and :math:`\sigma^\alpha` for
+    :math:`\alpha = x,y,z` are the Pauli matrices. For hardware upload, time is expected to be in units
+    of :math:`\mu \text{s}`, and the frequency in units of :math:`MHz`. It is recommended to also follow
```

Comment:
```suggestion
    of :math:`\text{µs}`, and the frequency in units of :math:`\text{MHz}`. It is recommended to also follow
```
Alternatively just MHz in text form (then it wouldnt be bold)